cmd/connect_test.go

```diff
@@ -2,51 +2,42 @@ package cmd
 
 import (
 	"context"
-	"os"
-	"path/filepath"
 	"testing"
 	"time"
 
-	"github.com/qri-io/ioes"
 	regmock "github.com/qri-io/qri/registry/regserver"
-	repotest "github.com/qri-io/qri/repo/test"
 )
 
 func TestConnect(t *testing.T) {
 	if err := confirmQriNotRunning(); err != nil {
 		t.Skip(err.Error())
 	}
 
+	// Setup the test repo so that connect can run
+	run := NewTestRunner(t, "test_peer", "qri_test_connect")
+	defer run.Delete()
+
+	// Construct a mock registry to pass to the connect command
 	_, registryServer := regmock.NewMockServer()
 
-	path := filepath.Join(os.TempDir(), "qri_test_commands_connect")
-	t.Logf("temp path: %s", path)
+	// Configure ports such that other tests do not conflict with the connection ports
+	// TODO(dustmop): Add websocket.port to config, set that here
```

Comment:
there's a partial-implementation of this in https://github.com/qri-io/qri/pull/1076, I'd love to pull off the `--no-p2p` part of that PR & revisit it.

Reply:
I'll leave the TODO for now, but will definitely look into that to fix websockets soon.